src/rockstor/fs/btrfs.py

```diff
@@ -20,6 +20,7 @@
 import re
 import time
 import os
+import shutil
```

Comment:
Is it still needed here? I cannot seem to find a use for it here.

Reply:
Ah, yes. I was trying to use something from there at first, before I found out about parse version. I'll get that removed.